### PR TITLE
Implement JWT management API

### PR DESCRIPTION
**Pull request by itaihanski**
## Related Issues

https://github.com/descope/etc/issues/5810

## Description

Implement JWT management API

## Must

- [x] Tests
- [x] Documentation (if applicable)

### CODE REVIEW

File: Descope.Test/IntegrationTests/Authentication/AuthenticationTests.cs [hunk 1]
```diff
@@ -12,7 +12,6 @@ public async Task Authentication_ValidateAndRefresh()
             string? loginId = null;
             try
             {
-                await _descopeClient.Management.User.DeleteAllTestUsers();
```
Comment (aviadl) [inline Descope.Test/IntegrationTests/Authentication/AuthenticationTests.cs:15]: Why did you remove this ? 
Are you cleaning up in a different place?

File: Descope.Test/IntegrationTests/Management/JwtTests.cs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                    Phone = "+972666666666",
+                });
+                var userId2 = response.UserId;
+
+                // Have user1 impersonate user2
+                var jwt = await _descopeClient.Management.Jwt.Impersonate(userId1, loginId2);
+
+                // Make sure the session is valid
+                var token = await _descopeClient.Auth.ValidateSession(jwt);
+                Assert.Equal(userId2, token.Id);
```
Comment (aviadl) [inline Descope.Test/IntegrationTests/Management/JwtTests.cs:70]: Want also to validate that you have the actor on the jwt ?

**Review by aviadl — COMMENTED** [review]
Few minor comments
Also dont we want to update readme ?

File: Descope.Test/IntegrationTests/Authentication/AuthenticationTests.cs (hunk 1 above)
Comment (itaihanski) [reply Descope.Test/IntegrationTests/Authentication/AuthenticationTests.cs:15]: Yes. Every test cleans up after itself.

**Review by aviadl — APPROVED** [review]
